### PR TITLE
Added info for EmailAddress DataType

### PR DESCRIPTION
**Pull request by bgunnarsson**
Added information for EmailAddress DataType.

For:
https://github.com/umbraco/UmbracoDocs/issues/1528

### CODE REVIEW

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+`Returns: String`
+
+Displays an email address.
+
+## Settings
+
+### Enable required checkbox (8.7.0 and below)
+
+If you type an email thats not correct the Umbraco will display a warning label under the property editor. When you publish the page that this editor is located on, Umbraco will clean the input.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md:17]: I'm not entirely sure I understand what is meant by this part here.
Could you perhaps elaborate a bit, and try to make a little clearer?

```suggestion
If you type an email that's not correct Umbraco will display a warning label under the property editor. When you publish the page that this editor is located on, Umbraco will clean the input.
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+If enabled, umbraco will display a warning label under the property editor when you publish the page this editor is located on and clean the input.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md:19]:
```suggestion
If the property is set to mandatory, Umbraco will display a warning label under the property editor when you publish the page this editor is located on and clean the input.
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+This functionality has been deprecated in `8.8.0` and above. You simply select the mandatory checkbox when adding the property editor to a document type.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md:21]:
```suggestion
This functionality has been deprecated in `8.8.0` and above. Instead, you select the mandatory checkbox when adding the property editor to a Document Type.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for adding this article @bgunnarsson !

I've made a few suggestions - mind taking a loot?

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+versionFrom: 8.0.0
+---
+
+# Dropdown
```
[the remark this answers is not in the document]
Comment (bgunnarsson) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md:5]: Yes, I'll take a look.

**Comment by bgunnarsson** [comment]
I pushed a new commit following your suggestions.

**Comment by sofietoft** [comment]
Aweoms! Thanks @bgunnarsson 💪

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+## MVC View Example
+
+### Typed
+
+```csharp
+@if (Model.HasValue("email"))
+{
+    var emailAddress = Model.Value<string>("email");
+
+    <p>@email</p>
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md:43]: Should be the same as the variable created in line 41.

```suggestion
    <p>@emailAddess</p>
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/index.md [hunk 6]
```diff
@@ -54,7 +54,7 @@ A configurable number control allowing only numbers including decimals.
 
 Displays a list of preset values. The content editor can select either a single or multiple values.
 
-## Email address
+## [Email address](Email-Address/index.md)
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/index.md:57]:
```suggestion
## [Email address](Email-Address)
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
I've added a few more suggestion - please take a look when you have a minute our two 💪

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md (hunk 5 above)
Comment (bgunnarsson) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Email-Address/index.md:43]: Whoops 😅

**Comment by bgunnarsson** [comment]
@sofietoft done and done 👍

**Review by sofietoft — APPROVED** [review]
Awesome! Thanks for applying the suggestions @bgunnarsson !

Let's get this merged in then 😄 
I can see this is your first-ever contribution to the Umbraco Docs - that's fantastic! 🙌 
Do you have an account on [Our](https://our.umbraco.com/)? If you do, I can grant you a C-Trib badge for your efforts. Let me know 😄

**Comment by bgunnarsson** [comment]
@sofietoft Thank you! First of many hopefully :D Yes this is my first contribution to the docs but I've had 2 minor contributions to the core before :)

My our profile:
https://our.umbraco.com/members/bgunnarsson/

**Comment by sofietoft** [comment]
That sounds awesome 😎 💪 

And you now have a C-Trib badge on your Our account.
Thanks again for helping out! Much appreciated 😄 
Cheers!

**Pull request merged**